main.css

```diff
@@ -0,0 +1,228 @@
+body {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.container{
+  display: grid;
+  grid-template: auto 2fr 2fr 2fr 2fr 2fr 3.5fr 2fr auto / 2fr 1fr 1fr;
+  grid-template-areas: 
+  "n n n"
+  "t t t"
+  "q1 q1 q1"
+  "w w w"
+  "w w w"
+  "q2 q2 q2"
+  "team team team"
+  "s s s"
+  "f f f";
+}
+
+.nav {
+  grid-area: n; 
+  margin: 20px;
+  flex-wrap: wrap;
+}
+.nav ul {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  list-style-type: none;
+}
+
+.try {
+text-align: right;
+color: #B39662;
+padding-left: 380px;
+}
+
+.about {
+  padding-right: 80px;
+
+}
+
+.tagline {
+  grid-area: t;
+  text-align: center;
+  font-weight: bold;
+  background-color: #BFFFDD;
+  border-bottom: solid;
+  border-width: 1px;
+}
+
+#startrly {
+  font-family: 'Heebo', sans-serif;
+  font-size: 4rem;
+  font-weight: bold;
+  letter-spacing: 1.4px;
+  
+  
+  
+}
+sub {
+  font-size: 1.5rem;
+  padding-bottom: 30px;
+  margin-top: 2px;
+}
+
+button {
+  border: 2px solid black;
+  border-radius: 10px;
+  text-align: center;
+  padding: 10px;
+  margin: 50px;
+  font-size: 2em;
+  }
+
+button:hover {
+  background-color:#FFE0A5;
+}
+a {
+  text-decoration: none;
+  color: #000;
+  padding: 15px;
+  
+}
+.quote1 {
+  grid-area: q1;
+  border-bottom: solid;
+  border-width: 1px;
+  margin-top: 10em;
+}
+
+blockquote {
+  margin: 1.5em 10px;
+  padding: 0.5em 10px;
+  quotes: "\201C""\201D""\2018""\2019";
+  font-style: italic;
+}
+
+blockquote:before {
+  color: #000;
+  content: open-quote;
+  font-size: 4em;
+  line-height: 0.1em;
+  margin-right: 0.25em;
+  vertical-align: -0.4em;
+}
+blockquote:after {
+  color: #000;
+  content: close-quote;
+  font-size: 4em;
+  line-height: 0.1em;
+  margin-right: 1em;
+  vertical-align: -0.5em;
+}
+
+.author {
+  text-align: right;
+  margin-right: 30px;
+  padding-right: 40px;
+  font-style: italic;
+}
+
+.why {
+  grid-area: w; 
+  /* max-height:30em; */
+  /* display: flex;
+  flex-direction: row; */
+  list-style: none;
+}
+.why h2 {
+  text-align: center;
+}
+.why ul {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+/* .why img {
+ 
+}
+.why #top {
+  
+  /* display: flex;
+  justify-content: center; }*/
+
+ /* .why li {
+  max-width:;
+  
+} */
+
+.why ul li{
+  display: flex;
+  list-style-type: none;
+  padding-right:  ; 
+  /* margin: 0;
+  padding: 0; */
+}
+  
+
+
+/* .why p {
+  width: 10em;
+} */
+
+.quote2 {
+  grid-area: q2;
+}
+
+.team {
+  grid-area: team;
+  flex-wrap: flex; 
+  flex-direction: row;
+  justify-content: center;
+  
+}
+
+/* .team h2 {
+  text-align: center;
+  padding-bottom: 10em;
+} */
+
+.team img {
+  border-radius: 50%;
+  
+}
+.first-three {
```

Comment:
I think if you put the flex-direction directly within the classes, it'll put the first 3 team-members on the first row and the rest on the second row.